samcli/commands/local/cli_common/options.py

```diff
@@ -166,7 +166,8 @@ def invoke_common_options(f):
             click.option(
                 "--container-env-vars",
                 type=click.Path(exists=True),
-                help="JSON file containing environment variables to be set within the container environment",
+                help="JSON file containing environment variables to be set within the container "
+                "environment when used in a debugging session locally.",
```

Comment:
do we need to mention that these are extra environment variables and not a replacement for `--env-vars` parameter?

Reply:
Maybe something like this can help make this a little more clear:

`JSON file containing additional environment variables to be set within the container when used in a debugging session locally.`

Adding the word "additional" and removed "environment"